cabal-install/tests/IntegrationTests/custom/custom_dep.sh

```diff
@@ -0,0 +1,8 @@
+. ./common.sh
+cd custom_dep
+cabal sandbox init
+cabal sandbox add-source custom
+cabal sandbox add-source client
+# Set HOME to something goofy so that we force the user
```

Comment:
No, that does not work, because these settings don't apply to `Setup.hs` build.

```
$ ghc-pkg-7.6 list
/opt/ghc/7.6.3/lib/ghc-7.6.3/package.conf.d
   Cabal-1.16.0
/home/ezyang/.ghc/x86_64-linux-7.6.3/package.conf.d
   Cabal-1.22.3.0
   Cabal-1.23.0.0
   Cabal-1.23.1.0
   Cabal-1.23.2.0
$ cabal configure -v --package-db=clear --package-db=global -w ghc-7.6
...
/srv/prod/bin/ghc-7.6 --make -odir ./dist/setup -hidir ./dist/setup -i -i. -package-id Cabal-1.22.3.0-9afd7fdd87eadc07ff9f73c86e22d4fe ./dist/setup/setup.hs -o ./dist/setup/setup -threaded
```

I'll add a comment to this effect.